README.md

```diff
@@ -30,14 +30,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    Unlike .forEach, .map returns a new array while passing those elements to the callback function.
+
 2. What is the difference between a function and a method?
+    
+    Functions can hold values inside them while methods are ususally used to invoke functions.
 
 3. What is closure?
 
+    Closure refers to the lexical scope of a function.
```

Comment:
This is correct. Furthermore, A closure is a nested function that has access to its outer function's variables.  Closures store references to the outer function's variables even after function invocations have completed.